eng/testing/FindWasmHostExecutableAndRun.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
+
+source $SCRIPT_DIR/FindWasmHostExecutable.sh "$1"
+
+if [ -n "${WASM_HOST_EXECUTABLE}" ]; then
+  shift
+  $WASM_HOST_EXECUTABLE "$WASM_BINARY_TO_EXECUTE" $WASM_HOST_ARGS_SEPERATOR "$@"
```

Comment:
`.cmd` also logs the command it is about to run.

Reply:
added log, thanks.